projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-cell.component.html

```diff
@@ -0,0 +1,31 @@
+<ng-template #defaultCell>
```

Comment:
Is there a need for a new hierarchical-cell template? As the same checks for the displayPinnedChip and the same template for the additional chip will probably need to be added to tree grid cell, perhaps it will be best to have this in the base cell so it can be re-used.

Reply:
It is supported by both but the regular cells do not support it, so it will only pollute their template. Also the tree grid cell has a separate cell template so it will need to be added there anyway.